docs/functionality/motors.md

```diff
@@ -0,0 +1,135 @@
+# Motors interface under tfc
+## SpeedRatio
+A concept that
+maps a motors capable
+range from 1-100%
+
+### Stopping
+ (-1<=>1) is outside the motors
+ range by definition and
+ the motor shall be stopped
+ if a speed is applied in that range.
+
+### Rationalization
+We have a plethora of
+use cases where precise motor
```

Comment:
```suggestion
use cases where explicit motor
```

Reply:
To my understanding 'precise' is deduced from precision and the control of the motor is still precise even though the speed is set relative to lsp-hsp.
So I suggested 'explicit', meaning that you don't need to explicitly set the velocity (m/s) as it is relative.